addon/templates/components/sessions-grid.hbs

```diff
@@ -30,65 +27,85 @@
+                {{t "general.ilm"}}: {{fa-icon "user-clock"}}
+                {{t "general.duePriorTo"}}:
+              </b>
+              {{#link-to "session" obj.course.id obj.postrequisite.id}}
+                {{truncate-text obj.postrequisite.title max=18}}
+              {{/link-to}}
+            {{else}}
+              <b>{{t "general.ilm"}}: {{t "general.dueBy"}}</b>
+              {{moment-format (await obj.session.firstOfferingDate) "MM/DD/YYYY"}}
+            {{/if}}
           {{else}}
-            {{moment-format (await obj.session.firstOfferingDate) "L LT"}}
+            {{#if obj.postrequisite}}
+              <b>{{fa-icon "user-clock"}} {{t "general.duePriorTo"}}:</b>
+              {{#link-to "session" obj.course.id obj.postrequisite.id}}
+                {{truncate-text obj.postrequisite.title max=18}}
+              {{/link-to}}
+            {{else}}
+              {{moment-format (await obj.session.firstOfferingDate) "L LT"}}
+            {{/if}}
           {{/if}}
         {{/if}}
       </span>
       <span class="session-grid-offerings">{{obj.offeringCount}}</span>
       <span class="session-grid-status" data-test-status>
         {{#if (and obj.canUpdate obj.instructionalNotes.length)}}
-          {{fa-icon
-            "clipboard-list"
-            prefix="fal"
-            class="instructional-notes"
-            title=(t "general.instructionalNotes")
+          {{fa-icon "clipboard-list"
             ariaHidden=false
-          }}
+            class="instructional-notes"
+            prefix="fal"
+            title=(t "general.instructionalNotes")}}
         {{/if}}
         {{publication-status item=obj showText=false}}
       </span>
       <span class="session-grid-actions" data-test-actions>
         {{#if obj.canUpdate}}
-          {{fa-icon
-            "trash"
+          {{fa-icon "trash"
+            ariaHidden=false
             class="clickable remove enabled"
             title=(t "general.remove")
-            ariaHidden=false
-            click=(action "confirmDelete" obj.session.id)
-          }}
+            click=(action "confirmDelete" obj.session.id)}}
         {{else}}
-          {{fa-icon
-            "trash"
-            class="disabled"
-            title=(t "general.remove")
+          {{fa-icon "trash"
             ariaHidden=false
-          }}
+            class="disabled"
+            title=(t "general.remove")}}
         {{/if}}
       </span>
     </div>
+
     {{#if (contains obj.session.id confirmDeleteSessionIds)}}
       <div class="confirm-removal" data-test-confirm-removal>
         {{t "general.confirmSessionRemoval"}}
         <div class="confirm-buttons">
-          <button {{action (perform removeSession obj.session)}} class="cancel" disabled={{removeSession.isRunning}} data-test-yes>
+          <button
+            class="cancel"
+            data-test-yes
+            disabled={{removeSession.isRunning}}
+            onclick={{perform removeSession obj.session}}>
```

Comment:
If replacing `{{action}}` with `onclick` you also need to handle `onpress` and maybe other keyboard events. I think the change is good, but please investigate how to handle the user pressing `<Enter>` or `<Space>` on the kayboard while the button has focus and see what ember does automatically for us that now needs to be replaced.

Reply:
The action handler gets called properly when I focus on the button and then press `enter` or `space`. So no further changes needed here.